client/blocks/nav-unification-quick-switch-modal/style.scss

```diff
@@ -0,0 +1,241 @@
+// @TODO: remove the ignore rule and replace font sizes accordingly
+/* stylelint-disable scales/font-size */
+
+@import '@automattic/typography/styles/fonts';
+@import '@automattic/onboarding/styles/mixins';
+@import '@wordpress/base-styles/breakpoints';
+
+$modal-breakpoint: 600px;
+$modal-breakpoint-phone-height: 415px;
+$modal-padding-v: 40px;
+$modal-padding-h: 50px;
+$modal-content-min-height: 410px;
+$modal-content-min-height-landscape: 370px;
+$modal-footer-padding-v: 20px;
+$modal-footer-height: 30px + ( $modal-footer-padding-v * 2 );
+
+// Core modal style overrides
+.nav-unification-quick-switch-modal {
+	&.components-modal__frame {
+		overflow: visible;
+		height: 65vh;
+		top: calc( 17.5vh - #{$modal-footer-height / 2} );
+
+		@media ( max-width: $modal-breakpoint ) {
+			width: 90vw;
+			min-width: 90vw;
+			left: 5vw;
+			right: 5vw;
+		}
+
+		@media ( min-width: $modal-breakpoint ) {
+			width: 740px;
+			height: $modal-content-min-height;
+			top: calc( 50% - #{$modal-footer-height / 2} );
+		}
+
+		@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+			height: $modal-content-min-height-landscape;
+			top: calc( 50% - 0px );
+		}
+	}
+
+	* {
+		box-sizing: border-box;
+	}
+
+	.components-modal__header {
+		display: block;
+		position: fixed;
+		text-align: right;
+	}
+
+	.components-guide__footer {
+		display: none;
+	}
+
+	.components-guide__container {
+		margin-top: 0;
+	}
+
+	.components-guide__page {
+		position: absolute;
+		width: 100%;
+		justify-content: start;
+
+		@media ( min-width: $modal-breakpoint ) {
+			height: $modal-content-min-height;
+		}
+
+		@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+			height: $modal-content-min-height-landscape;
+		}
+	}
+
+	.components-guide__page-control {
+		position: relative;
+		height: 0;
+		top: 100%;
+		overflow: visible;
+		margin: 0 auto;
+		z-index: 2;
+
+		&::before {
+			display: inline-block;
+			content: '';
+			height: $modal-footer-height;
+			vertical-align: middle;
+		}
+
+		li {
+			vertical-align: middle;
+			margin-bottom: 0;
+		}
+
+		// Temporarily disable dots on mobile as alignment is wonky.
+		display: none;
+		@media ( min-width: $modal-breakpoint ) {
+			display: block;
+		}
+	}
+
+	.components-guide__finish-button {
```

Comment:
Same here, AFAIK this button is never visible so we can probably get rid of these styles.